src/routes.js

```diff
@@ -26,6 +27,7 @@ export const getRoutes = (store) => {
       <Route path="/signup" component={SignupContainer} />
       <Route path="/home" component={HomeContainer} onEnter={authRequired}/>
       <Route path="/login" component={LoginContainer} />
+      <Route path="/target" component={Target} />
```

Comment:
this route should not be added, I have to remove it because you could access the _create target_ view through the side bar panel.